### DISCUSSION TITLE
How to Change Colour of a button?

### DISCUSSION THREAD

**Discussion by arunim2405** [cloudscape-design/components#408] [Q&A] [answered]
I am trying to create a trash button with red colour, I do not want to change the whole theme a create a new theme for all the buttons, I just need a red button instead of the normal blue one, with css being hashed, any easy way to do this?

**Comment by vitalijalbu** [comment]
write inline css code style={{background: 'red'}}

**Reply by just-boris** [reply]
This is not supported either. Our components cannot be customized this way

**Comment by just-boris** [comment, marked as answer]
Hello!

Changing colors of individual buttons is not supported in our design. As our [color guidelines](https://cloudscape.design/foundation/visual-foundation/colors/#guidelines) say

> There are various ways colors can be used to convey messages; however, color should never be the only visual means of conveying information. Ensure users who cannot see color can still perceive information by using other visual means, such as pairing color with iconography or text. 

For the delete button in particular we suggest using a confirmation dialog. For more details, see this article: https://cloudscape.design/patterns/resource-management/delete/
